docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc

```diff
@@ -638,6 +638,70 @@ Avaya
 Avaya bought Nortel's wired networks assets. So Avaya switches are, in effect, re-branded 
 Nortels. See <<_nortel,Nortel section>> of this document for configuration instructions. 
 
+802.1X with MAC Authentication bypass and VoIP
+++++++++++++++++++++++++++++++++++++++++++++++
+
+Global config settings:
+
+  sntp server primary address 192.168.1.5
```

Comment:
In that case, we should explain that the NTP server should be installed on the server and to open the appropriate ports in iptables.conf